pkg/trainer/tensorboard.go

```diff
@@ -178,6 +178,8 @@ func (s *TBReplicaSet) getDeploymentSpecTemplate(image string) v1.PodTemplateSpe
 
 	ps.Volumes = append(ps.Volumes, s.Spec.Volumes...)
 
+	ps.SchedulerName = s.Job.SchedulerName()
```

Comment:
There is no TensorBoard now so we could remove the changes here.

Reply:
OK, I'll remove it in the next update. Thanks.